less/index.less

```diff
@@ -48,129 +50,210 @@ table {
+            width: 90%;
+        }
+        @media (max-width: 400px) {
+            width: 100%;
+        }
+    }
+    nav ul {
+        margin-top: 40px; 
+        width: 610px;
+        justify-content: space-between;
+        display: flex;
+        font-size: 1.6rem;
+        @media (max-width: 768px) {
+            width: 100%;
+            margin: 20px 0;
+        }
+        @media (max-width: 400px) {
+            display: flex;
+            flex-direction: column;
+            margin: 0;
+            align-items: center;
+        }
+        li {
+            @media (max-width: 400px) {
+                width: 100%;
+                margin: 0;
+            }
+        }
+        li a {
+            text-decoration: none;
+            color: #b0b0b0;
+            @media (max-width: 400px) {
+                display: block;
+                text-align: center;
+                padding: 20px 0;
+                font-size: 2rem;
+                border-bottom: 1px solid gray;
+            }
+        }
+        li:first-child {
+            @media (max-width: 400px) {
+                border-top: 1px solid gray;
+            }
+        }
+        li a:hover {
+            text-decoration: underline;
+            color: #b0b0b0 / 1.3;
+            @media (max-width: 400px) {
+                background: gray;
+            }
+        }
+    }
+    .logo {
+        align-self: flex-end;
+        margin: 28px 0 0 91px;
+        @media (max-width: 768px) {
+            order: -1;
+            max-width: 100%;
+            margin: 20px 0;
+        }
+    }
 }
 
-header nav ul {
-    margin-top: 40px; 
-    width: 610px;
-    justify-content: space-between;
-    display: flex;
-}
-
-header nav ul li a {
-    text-decoration: none;
-    color: #b0b0b0;
-}
-
-header nav ul li a:hover {
-    text-decoration: underline;
-    color: black;
-}
-
-header .logo {
-    align-self: flex-end;
-    margin: 28px 0 0 91px;
-}
-
-/* Call To Action (CTA) Styles */
+// Call To Action (CTA) Styles
 
 .cta {
     display: flex;
     margin-top: 72px;
     justify-content: flex-end;
+    @media (max-width: 768px) {
+        margin-top: 50px;
+        flex-flow: column;
+        align-items: center;
+    }
+    .cta-text {
+        font-size: 7.2rem;
+        text-align: center;
+        letter-spacing: 5px;
+        display: flex;
+        justify-content: center;
+        flex-flow: column;
+        align-items: center;
+        margin: 0 82px 0 0;
+        @media (max-width: 768px) {
+            margin: 0;
+        }
+        button {
+            border: 1px double black;
+            font-size: 1.6rem;
+            background: white;
+            width: 85 + 85px;
+            height: 32px;
+            margin-top: 10px;
+            cursor: pointer;
+        }
+        button:hover {
+            background: black;
+            color: white;
+        }
+    }
+    img {
+        @media (max-width: 768px) {
+            display: none;
+        }
+    }
 }
 
```

Comment:
Nice nesting.
One thing I would suggest is to add comments to the end of the selectors so that you're never in a situation where you are counting curly braces to find where a selector ends.